main.cpp

```diff
@@ -35,7 +37,18 @@ int main(int argc, char *argv[])
     std::cout << Text::bold(Text::center("Made with 💛 by Maracatronics Robotics Team.")) << '\n';
     std::cout << Text::bold(Text::center("Version " + a.applicationVersion().toStdString())) << '\n' << '\n';
 
+    // Setup ExitHandler
+    ExitHandler::setApplication(&a);
```

Comment:
With this, ExitHandler is capable of know what application it will see and setup every steps for it